packages/@okta/vuepress-site/conductor.yml

```diff
@@ -2793,3 +2793,51 @@ redirects:
     to: /docs/guides/configure-signon-policy/main/#prompt-for-an-mfa-factor-when-a-user-is-outside-the-us
   - from: /docs/guides/configure-signon-policy/next-steps/index.html
     to: /docs/guides/configure-signon-policy/main/#next-steps
+  - from: /guides/sign-into-web-app/-/before-you-begin/index.html
+    to: /docs/guides/sign-into-web-app/-/before-you-begin/
+  - from: /guides/sign-into-web-app/-/configure-packages/index.html
+    to: /docs/guides/sign-into-web-app/-/configure-packages/
+  - from: /guides/sign-into-web-app/-/create-okta-application/index.html
+    to: /docs/guides/sign-into-web-app/-/create-okta-application/
+  - from: /guides/sign-into-web-app/-/define-callback/index.html
+    to: /docs/guides/sign-into-web-app/-/define-callback/
+  - from: /guides/sign-into-web-app/-/get-user-info/index.html
+    to: /docs/guides/sign-into-web-app/-/get-user-info/
+  - from: /guides/sign-into-web-app/-/next-steps/index.html
+    to: /docs/guides/sign-into-web-app/-/next-steps/
+  - from: /guides/sign-into-web-app/-/redirect-to-sign-in/index.html
+    to: /docs/guides/sign-into-web-app/-/redirect-to-sign-in/
+  - from: /guides/sign-into-web-app/-/require-authentication/index.html
+    to: /docs/guides/sign-into-web-app/-/require-authentication/
```

Comment:
It appears all the "to:" redirects are incorrect. To go to a heading within the single page, we need to add the anchor. For example:

`from: /docs/guides/sign-into-web-app/-/define-callback/index.html`
`to: /docs/guides/sign-into-web-app/main/#understand-the-callback-route`
`from: /docs/guides/sign-into-web-app/-/define-callback`
`to: /docs/guides/sign-into-web-app/main/#understand-the-callback-route`

Reply:
You need the generic stack selector in the `to` clause (if there are stack selectors): 
```
from: /docs/guides/sign-into-web-app/-/define-callback/index.html
to: /docs/guides/sign-into-web-app/-/main/#understand-the-callback-route
from: /docs/guides/sign-into-web-app/-/define-callback
to: /docs/guides/sign-into-web-app/-/main/#understand-the-callback-route
```

Reply:
Updated my example above as per Van's comment. Thanks, Van!